lambdas/services/document_manifest_service.py

```diff
@@ -1,104 +1,104 @@
-import logging

```

Comment:
Interesting it's happened on only two files in the PR. I have my IDE set to LF endings so shouldn't be an issue with line endings. It seems some people might be forgetting to format before pushing so there's lot's of files that have been changed so that might be the reason?

Reply:
Seems there was a number of CRLF AND LF files in the repo, if you run `git ls-files --eol` you'll be able to see them. The reason the file had changed in my PR is I've changed it from CRLF to LF as you can see in the screenshot 
![image](https://github.com/nhsconnect/national-document-repository/assets/127417949/cf368adf-d60e-4384-8f07-39718c3738b3)

Reply:
I'm going to change the rest of them as part of this PR so we have LF going forward, I'm not sure if this may have come from your machine as you were switching between Windows and WSL? I'll wait till the PR is completely looked over however so it's not difficult for people to locate the actual changes

Reply:
Might have been me, I've been super careful to try to avoid this but windows CRLFs are a fickle beast.
Happy with these changes, thanks Abbas